src/Entity/FileEntity.php

```diff
@@ -228,24 +229,35 @@ protected function updateImageFieldDimensions() {
 
         $entities = \Drupal::entityTypeManager()
           ->getStorage($entity_type_id)
-          ->load($ids);
-          /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
+          ->loadMultiple($ids);
+
         foreach ($entities as $entity) {
-          foreach (array_keys($entity->getTranslationLanguages()) as $langcode) {
-            $translation = $entity->getTranslation($langcode);
-            foreach ($translation->$image_field as $item) {
-              if ($item->target_id == $this->id()) {
-                $item->width  = $this->getMetadata('width');
-                $item->height  = $this->getMetadata('height');
-              }
-            }
-          }
+          $this->updateImageFieldDimensionsByEntity($entity, $image_field);
+        }
+      }
+    }
+  }
 
-          // Save the updated field column values.
-          $entity->save();
+  /**
+   * @param \Drupal\Core\Entity\ContentEntityInterface $entity
```

Comment:
A docblock needs an single line description that explains what the function does, usually starts with a verb like Updates ...